airbyte-integrations/connectors/destination-bigquery/README.md

```diff
@@ -16,6 +16,7 @@ Note that the `secrets` directory is git-ignored by default, so there is no dang
 
 #### Build
 Build the connector image via Gradle:
+You need a local install of [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) to run this command.
```

Comment:
You don't. Gradle will provision a python venv and do the poetry song-and-dance to install airbyte-ci from source in the .venv folder in the repo root.

Reply:
Oh yes, I forgot your wizardries!